client/src/cb-admin/pages/VisitorDetails.js

```diff
@@ -106,15 +122,27 @@ export default class VisitorDetailsPage extends React.Component {
 
         this.setState({ users: res.data.result });
       })
-      .catch((error) => {
-        if (ErrorUtils.errorStatusEquals(error, 401)) {
-          this.props.history.push('/admin/login');
-        } else if (ErrorUtils.errorStatusEquals(error, 500)) {
-          this.props.history.push('/internalServerError');
-        } else {
-          this.setState({ errors: { general: 'Could not update activity' } });
-        }
-      });
+      .catch(error => console.log(error));
+
+    let yearRange = [];
```

Comment:
@RogeredBacon weird that this is coming up in your dif. This was removed in my last merged PR. Did you copy and paste this file at any point?

Reply:
Ah it'll be from the conflicts when I rebased on master. I'll fixy

Reply:
@astroash I think I've fixed it. Would you mind taking a look and seeing if everything is as it should be, then I'll squash the commits.